my.css

```diff
@@ -0,0 +1,107 @@
+.main {
+	position: relative;
+	max-width: 1200px;
+	height: 100%;
+	
+}
+.burgermenu {
+	position: absolute;
+	right: 0;
+	top: 0;
+	width: 70px;
+	height: 70px;
+	background-color: #e3e1e1;
+
+}
+.burger {
+	position: absolute;
+	top: 20px;
+    left: 18px;
+	width: 40px;
+    height: 4px;
+    margin-top: 5px;
+    background: #000;	
+}
+.burger::before {
+	content: '';
+	position: absolute;
+	width: 40px;
+	top: 13px;
+	left: 0;
+	height: 5px;
+	background: #000;
+}
+.promo {
+    width: 1440px;
+    height: 900px;
+    margin: 0 auto;
+    background: -webkit-linear-gradient(180deg, white 70%, black 30%);
+}
+.header__promo {
+	width: 30%;
+	display: flex;
+	justify-content: space-between;
+	
+}
+.logo__promo {
+	padding-top: 20px;
+    padding-left: 20px;
+	color: #fff;
+	font-family: Arial, sans-serif, Montserrat;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 25px;
+  
+
+}
+.ru__promo {
+    padding-top: 30px;
+    padding-right: 20px;
+}
+.ru__link {
+	text-decoration: none;
+	color: #fff;
+	font-family: Arial, sans-serif, Montserrat;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+}
+.collection {
+	margin: 100px;
+    display: flex;
+    justify-content: center;
+}
+.newcollection {
+	display: flex;
+    flex-direction: column;
+    width: 40%;
+    padding: 30px;
+    margin-left: 20px;
+    justify-content: space-between;
+}
+.newcollection__text {
+    height: 40%;
+    font-family: Arial, sans-serif, Montserrat;
+    font-style: normal;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)